code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -858,6 +858,9 @@ GLOBAL_LIST_EMPTY(PDAs)
 			else if (istype(A, /obj/machinery/atmospherics/pipe))
 				var/obj/machinery/atmospherics/pipe/P = A
 				atmosanalyzer_scan(P.parent.air, user, P)
+			else if (istype(A, /obj/machinery/atmospherics/components/unary))

```

Comment:
This will not mean you can scan cryotubes and other such machines with a unary device within the machine. It will check if the machine you are interacting with is a unary device,

Reply:
cryo tubes are descendants of /obj/machinery/atmospherics/components/unary, so yes that is what this means.
they don't have a unary device -inside- the machine, they are the unary device